yt_dlp/extractor/canalsurmas.py

```diff
@@ -0,0 +1,143 @@
+import json
+
+from .common import InfoExtractor
+from ..networking.common import Request
+
+
+class CanalsurmasExtractorIE(InfoExtractor):
+    _VALID_URL = r'https://www.canalsurmas.es/videos/(?P<id>[0-9]*)-.*'
+
+    _TESTS = [
+        {
+            'url': 'https://www.canalsurmas.es/videos/44006-el-gran-queo-1-lora-del-rio-sevilla-20072014',
+            'md5': '861f86fdc1221175e15523047d0087ef',
+            'info_dict': {
+                'id': '44006',
+                'ext': 'mp4',
+                'title': 'Lora del Río (Sevilla)  ',
+                'description': 'md5:3d9ee40a9b1b26ed8259e6b71ed27b8b',
+            },
+        },
+    ]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+
+        access_token = self._download_json(
+            Request(
+                url='https://api-rtva.interactvty.com/jwt/token/',
+                method='POST',
+                headers={
+                    'Content-Type': 'application/json',
+                },
+                data=json.dumps(
+                    {
+                        'username': 'canalsur_demo',
+                        'password': 'dsUBXUcI',
+                    },
+                ).encode(),
+            ),
+            video_id,
+        )['access']
+
+        # An access token must be sent along with this request, otherwise, the server will return an unauthorized response. The access token can be obtained via the following global variable: window.__NUXT__.state.auth.accessToken (in the context of the video webpage).
+        '''
+        The 'optional_fields' query parameter of this request can have 1 or more values delimited by a ',' (comma) character. The known valid values for this query parameter are:
```

Comment:
No need to document all optional fields. Just request and use the ones that make sense to be included as part of the info dict. https://github.com/yt-dlp/yt-dlp/blob/master/yt_dlp/extractor/common.py#L120

image, duration, genre, created_at, publish_date, and tags seem like good things to include.